### PR TITLE
Reorganizing repository structure

### PR DESCRIPTION
**Pull request by ywkim312**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,3 +23,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Created base image for sentiment analysis with model [#55](https://github.com/ncsa/standalone-smm-analytics/issues/55)
 - Created base image for name entity recognition with model [#56](https://github.com/ncsa/standalone-smm-analytics/issues/56)
 - Docker compose file updated to fix minio default bucket making [#63](https://github.com/ncsa/standalone-smm-analytics/issues/63)
+- Reorganized repository structure [#10](https://github.com/ncsa/standalone-smm-analytics/issues/10)
```
Comment (longshuicy) [inline CHANGELOG.md:26]: Can you start a Unrelease session. Last release is Beta release, i'm thinking of proper tagging it from now on

File: batch/batch_network_analysis/algorithm.py [hunk 2]
```diff
@@ -1,37 +0,0 @@
-import plot
```
Comment (longshuicy) [inline batch/batch_network_analysis/algorithm.py:1]: what happened to this file?

File: batch/batch_preprocessing/algorithm.py [hunk 3]
```diff
@@ -1,38 +0,0 @@
-import plot
```
Comment (longshuicy) [inline batch/batch_preprocessing/algorithm.py:1]: Could you also see why this file content disappears? Theoretically this PR should only has name change

File: batch/batch_network_analysis/algorithm.py (hunk 2 above)
Comment (ywkim312) [reply batch/batch_network_analysis/algorithm.py:1]: that file still in the aws/batch/batch_network_analysis folder
image

File: batch/batch_preprocessing/algorithm.py (hunk 3 above)
Comment (ywkim312) [reply batch/batch_preprocessing/algorithm.py:1]: it should be still there. 

image

File: CHANGELOG.md (hunk 1 above)
Comment (ywkim312) [reply CHANGELOG.md:26]: oh, didn't know. sorry, updated

**Comment by longshuicy** [comment]
@ywkim312 but i see a lot of file rename from batch to lambda. e.g. 
image

**Comment by longshuicy** [comment]
I

> @ywkim312 but i see a lot of file rename from batch to lambda. e.g. image

Sorry, im just being cautious that we don't lose any information

**Comment by ywkim312** [comment]
> I
> 
> > @ywkim312 but i see a lot of file rename from batch to lambda. e.g. image
> 
> Sorry, im just being cautious that we don't lose any information

It is so weird. It looks okay in the actual files. Let's see how it looks like when it gets merged

**Comment by ywkim312** [comment]
I will redo this in another PR

**Pull request closed without merging**